test/unit/spec/EnrollSms_spec.js

```diff
@@ -214,6 +214,15 @@ Expect.describe('EnrollSms', function() {
         expect(test.form.getCodeFieldAutocomplete()).toBe('off');
       });
     });
+    itp('has search input display set correctly', function() {
+      return setup().then(function(test) {
+        const searchInput = $sandbox.find('.chzn-search > input');
+        expect(searchInput.css('display')).toBe('none');
+        const mousedown = $.Event('mousedown');
+        test.form.countryDropdown().trigger(mousedown);
+        expect(searchInput.css('display')).toBe('inline-block');
+      });
+    });
```

Comment:
This test case could be split up so the test titles are more informative, like 'has a hidden search input by default' and 'shows search input on mousedown'

Reply:
Good point, I split up the test.